<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#9ec2e6;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1021</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>401</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;color:#fff;
font-family: Arial;
font-size: 58px;
font-weight:700;
</string>
      </property>
      <property name="text">
       <string>ShopManager</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>490</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: rgba(0,0,0,0.5);</string>
      </property>
      <property name="text">
       <string>--------------------------Hoặc-------------------------</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>150</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
font-family: Arial;
font-size: 42px;
font-weight:700;
color:rgb(12, 5, 230)</string>
      </property>
      <property name="text">
       <string>Chào Mừng</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="inforSV">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>430</y>
        <width>471</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;color:#fff;
border: 0px;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;PBL2: Lập Trình Hướng Đối Tượng&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:16pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Sinh viên:     Nguyễn Quốc Duy         - 102230289&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Sinh viên:     Trần Tâm Như                - 102230311&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;GVHD: 	       Mai Văn Hà&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>390</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>51</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <underline>true</underline>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color:rgb(38, 49, 255) ;
font-family: Arial;
</string>
      </property>
      <property name="text">
       <string>Quên mật khẩu?</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>290</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #e8f0fd;
border-radius: 2px;
border: 1px solid #000;</string>
      </property>
      <property name="placeholderText">
       <string>Nhập ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>240</y>
        <width>171</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resourse.qrc">:/new/img/img/images.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>430</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#4561ff;
font-weight: 700;
color: white;
</string>
      </property>
      <property name="text">
       <string>Đăng Nhập</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>200</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>51</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <fontweight>Thin</fontweight>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
font-family: Arial;
font-size: 32px;
font-weight5700;
</string>
      </property>
      <property name="text">
       <string>trở lại!</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pw">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>340</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #e8f0fd;
border-radius: 2px;
border: 1px solid #000;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Nhập Mật Khẩu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>60</y>
        <width>451</width>
        <height>621</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#fff;
border-radius: 30px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>530</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
font-family: Arial;
font-size: 18px;
font-weight:500;</string>
      </property>
      <property name="text">
       <string>Chưa có tài khoản? </string>
      </property>
     </widget>
     <widget class="QPushButton" name="closeBtn">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>430</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ccc;</string>
      </property>
      <property name="text">
       <string>Thoát</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>511</width>
        <height>621</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:qconicalgradient(cx:0, cy:0.022, angle:217.1, stop:0 rgba(0, 18, 180, 255), stop:1 rgba(255, 255, 255, 255));
border-radius: 30px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="loginStatus">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>390</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;color:red;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="signUpBtn">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>530</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color:rgb(38, 49, 255) ;
font-family: Arial;
font-size: 18px;

border: none;</string>
      </property>
      <property name="text">
       <string>Đăng Ký</string>
      </property>
     </widget>
     <zorder>label_5</zorder>
     <zorder>label_4</zorder>
     <zorder>label_9</zorder>
     <zorder>label_2</zorder>
     <zorder>label</zorder>
     <zorder>inforSV</zorder>
     <zorder>label_3</zorder>
     <zorder>id</zorder>
     <zorder>label_10</zorder>
     <zorder>pushButton</zorder>
     <zorder>label_8</zorder>
     <zorder>pw</zorder>
     <zorder>label_6</zorder>
     <zorder>closeBtn</zorder>
     <zorder>loginStatus</zorder>
     <zorder>signUpBtn</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="id_2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>310</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #e8f0fd;
border-radius: 2px;
border: 1px solid #000;
padding:5px;</string>
      </property>
      <property name="placeholderText">
       <string>Số điện thoại</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pw_2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>360</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #e8f0fd;
border-radius: 2px;
border: 1px solid #000;
padding:5px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>530</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: rgba(0,0,0,0.5);</string>
      </property>
      <property name="text">
       <string>--------------------------Hoặc-------------------------</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="inforSV_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>430</y>
        <width>471</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;color:#fff;
border: 0px;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;PBL2: Lập Trình Hướng Đối Tượng&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:16pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Sinh viên:     Nguyễn Quốc Duy         - 102230289&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Sinh viên:     Trần Tâm Như                - 102230311&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;GVHD: 	       Mai Văn Hà&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>511</width>
        <height>621</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:qconicalgradient(cx:0, cy:0.022, angle:217.1, stop:0 rgba(0, 18, 180, 255), stop:1 rgba(255, 255, 255, 255));
border-radius: 30px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>130</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
font-family: Arial;
font-size: 42px;
font-weight:700;
color:rgb(12, 5, 230)</string>
      </property>
      <property name="text">
       <string>Đăng ký</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closeBtn_2">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>480</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ccc;</string>
      </property>
      <property name="text">
       <string>Thoát</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>240</y>
        <width>171</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resourse.qrc">:/new/img/img/images.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>480</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#4561ff;
font-weight: 700;
color: white;
</string>
      </property>
      <property name="text">
       <string>Đăng Ký</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_3">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>260</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #e8f0fd;
border-radius: 2px;
border: 1px solid #000;
padding:5px;</string>
      </property>
      <property name="placeholderText">
       <string>Họ và tên</string>
      </property>
     </widget>
     <widget class="QLabel" name="loginStatus_2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>450</y>
        <width>221</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;color:red;</string>
      </property>
      <property name="text">
       <string>Các trường không được để Trống</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>180</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
font-family: Arial;
font-size: 22px;

</string>
      </property>
      <property name="text">
       <string>Khách Hàng thường xuyên</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>401</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;color:#fff;
font-family: Arial;
font-size: 58px;
font-weight:700;
</string>
      </property>
      <property name="text">
       <string>ShopManager</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_4">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>310</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #e8f0fd;
border-radius: 2px;
border: 1px solid #000;
padding:5px;</string>
      </property>
      <property name="placeholderText">
       <string>Địa chỉ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>60</y>
        <width>451</width>
        <height>621</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#fff;
border-radius: 30px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>570</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
font-family: Arial;
font-size: 18px;
font-weight:500;</string>
      </property>
      <property name="text">
       <string>Đã có tài khoản? </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pw_3">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>410</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #e8f0fd;
border-radius: 2px;
border: 1px solid #000;
padding:5px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Mật khẩu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="returnLoginBtn">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>570</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color:rgb(38, 49, 255) ;
font-family: Arial;
font-size: 18px;

border: none;</string>
      </property>
      <property name="text">
       <string>Đăng Nhập</string>
      </property>
     </widget>
     <zorder>label_18</zorder>
     <zorder>label_13</zorder>
     <zorder>id_2</zorder>
     <zorder>pw_2</zorder>
     <zorder>label_12</zorder>
     <zorder>inforSV_2</zorder>
     <zorder>label_14</zorder>
     <zorder>closeBtn_2</zorder>
     <zorder>label_15</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>id_3</zorder>
     <zorder>loginStatus_2</zorder>
     <zorder>label_16</zorder>
     <zorder>label_17</zorder>
     <zorder>id_4</zorder>
     <zorder>label_19</zorder>
     <zorder>pw_3</zorder>
     <zorder>returnLoginBtn</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>id</tabstop>
  <tabstop>pw</tabstop>
  <tabstop>id_3</tabstop>
  <tabstop>id_2</tabstop>
  <tabstop>id_4</tabstop>
  <tabstop>pw_2</tabstop>
  <tabstop>pw_3</tabstop>
  <tabstop>closeBtn_2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>returnLoginBtn</tabstop>
  <tabstop>closeBtn</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>signUpBtn</tabstop>
  <tabstop>inforSV</tabstop>
  <tabstop>inforSV_2</tabstop>
 </tabstops>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
